route.php

```diff
@@ -459,6 +483,12 @@
   Router::route('/student/profile', 'student/profile');
 
   Router::route('/stats', 'stats');
+  Router::route('/stats/recruiters', 'stats/recruiters');
```

Comment:
Just for sake of identifying which routes are ajax routes, this (and below) should be /stats/ajax/recruiters. I will go ahead and fix this.